c/tree.c

```diff
@@ -1467,31 +1467,27 @@ LOCAL void assign_node_number(MMDBW_tree_s *tree, MMDBW_node_s *node,
 void freeze_tree(MMDBW_tree_s *tree, char *filename, char *frozen_params,
                  size_t frozen_params_size)
 {
-#ifdef WIN32
-    int fd = open(filename, O_CREAT | O_TRUNC | O_RDWR);
-#else
-    int fd = open(filename, O_CREAT | O_TRUNC | O_RDWR, (mode_t)0644);
-#endif
-    if (fd == -1) {
+    FILE *file = fopen(filename, "wb");
```

Comment:
Interesting that it was opened for reading as well as writing previously.

Reply:
Yeah, I am not sure what that was about.